examples/pwa/pwa-ampdoc-shell.html

```diff
@@ -0,0 +1,141 @@
+      window.AMP_CONFIG = {
+        'ampdoc-shell': 1
+      }
+    }
+  </script>
+
+  <style amp-custom>
+    body {
+      margin: 0;
+      font-family: Roboto, Arial;
+    }
+
+    header {
+      position: fixed;
+      top: 0;
+      left: 0;
+      right: 0;
+      height: 56px;
+      background: rgba(0, 0, 0, 0.9);
+      color: #eee;
+      display: flex;
+      align-content: center;
+      align-items: center;
+      justify-content: center;
+      font-size: 18px;
+    }
+
+    .container {
+      min-height: 100vh;
+      background: #eee;
+      overflow: hidden;
+    }
+
+    .container {
+      margin: 64px 8px;
+    }
+
+    .card {
+      background: #fff;
+      box-shadow: 0 1px 4px 0 rgba(0,0,0,0.14);
+      padding: 8px;
+      margin: 8px 0;
+    }
+
+    .card a {
+      text-decoration: none;
+    }
+
+    .card h4 {
+      margin: 8px 0;
+      font-size: 16px;
+      color: rgba(0,0,0,0.95);
+    }
+
+    .card .detail {
+      margin: 8px 0;
+      font-size: 14px;
+      color: rgba(0,0,0,0.87);
+    }
+
+    .amp-slot {
+      padding: 16px;
+      font-size: larger;
+      background: cyan;
+    }
+
+    .hamburger {
+      color: white;
+      font-size: 32px;
+      position: absolute;
+      left: 0;
+    }
+
+    .sidebarClose {
+      font-size: 32px;
+    }
+  </style>
+</head>
+<body>
+  <amp-sidebar id="sidebar" class="sidebar" layout="nodisplay" side="left">
+    <button id="sidebarClose" class="sidebarClose" on='tap:sidebar.close'>X</button>
+    <section id="stream" class="stream">
+      <article class="card">
+        <a href="/pwa/examples/article.amp.html">
+          <h4>Lorem Ipsum</h4>
+          <div class="detail">Fusce pretium tempor justo, vitae consequat dolor maximus eget.</div>
+        </a>
+      </article>
+      <article class="card">
+        <a href="/pwa/examples/youtube.amp.html">
+          <h4>YouTube video</h4>
+          <div class="detail">Fusce pretium tempor justo, vitae consequat dolor maximus eget.</div>
+        </a>
+      </article>
+      <article class="card">
+        <a href="/pwa/examples/analytics.amp.html">
+          <h4>Analytics</h4>
+          <div class="detail">Fusce pretium tempor justo, vitae consequat dolor maximus eget.</div>
+        </a>
+      </article>
+    </section>
+  </amp-sidebar>
+
+  <header>
+    <button class="hamburger" on='tap:sidebar.toggle'>☰</button>
+
+    PWA Chronicles
+  </header>
+
+  <div class="container">
+    <div id="doc-container" class="doc-container">
+      Document will appear here
+    </div>
+    <template id="amp-slot-template">
+      <div class="amp-slot" slot="slot1" onclick="alert('This is not allowed and I don\'t care!');">
+        This content comes from parent shell! Including forbidden elements, styles, etc.
+        <img src="/examples/img/hero@1x.jpg" style="width: 40px; height: 40px;">
+      </div>
+    </template>
+  </div>
+
+  <!-- Must be in the very end. -->
+  <script src="/pwa/pwa.js"></script>
```

Comment:
Very quickly: why must this be in the end? Asking b/c we are currently changing some of these examples to async include pwa.js and to include shadow-v0.js from the main doc itself independently with async as well. Just wanted to make sure this will not conflict with any of your plans.